lib/engine/game/g_1849/step/bond.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1849
+      module Step
+        class Bond < Engine::Step::Base
+          def actions(entity)
+            return [] if !entity.corporation? || entity != current_entity
+
+            actions = []
+            actions << 'payoff_loan' if can_payoff_loan?(entity)
+            actions << 'take_loan' if @game.can_take_loan?(entity)
+            actions << 'pass' if blocks? && !actions.empty?
+
+            actions
+          end
+
+          def description
+            can_payoff_loan?(current_entity) ? 'Repay Bond' : 'Issue Bond'
+          end
+
+          def log_skip(entity)
+            super if @game.bonds? && !@game.phase.status.include?('gray_uses_white')
```

Comment:
Why did you decide to gate by status instead of something like `@game.phase.previous.include? { |p| p.name == "6H" }` or an event or status that says bonds are available?